src/lib.rs

```diff
@@ -78,6 +78,7 @@ mod format;
 mod framebuffer;
 mod image;
 mod pipeline;
+mod pixels;
```

Comment:
Given the name change, `pixels` probably isn't the best module name. I thought of `attachments.rs` or `copy_attachments.rs`, or I could just move the methods into `framebuffer.rs`.

Reply:
The functions fit well into `framebuffer.rs`